NOTICE

```diff
@@ -0,0 +1,5 @@
+Apache NuttX
+Copyright 2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
```

Comment:
When the headers are updated to Apache ones (removing the copyright line) the relocated copyright should also be listed here.